config/repos.yml

```diff
@@ -115,3 +115,7 @@ repos:
   -
     link: 'https://github.com/IXix/JSFX'
     index: 'https://github.com/IXix/JSFX/raw/master/index.xml'
+  -   
+    link: 'https://github.com/ArchieScript/Archie_ReaScripts'
+    index: 'https://github.com/ArchieScript/Archie_ReaScripts/raw/master/index.xml' 
+  -
```

Comment:
Thank you, all good except for the extra hyphen in the last line. (It adds a null element to the repository list as per YAML syntax.)
```suggestion
```